src/controllers/users.controller.js

```diff
@@ -0,0 +1,84 @@
+'use strict';
+
+const userService = require('../services/users.services');
+
+const getAll = (req, res) => {
+  res.send(userService.getAll());
+};
+
+const getById = (req, res) => {
+  const { id } = req.params;
+
+  const user = userService.getById(+id);
+
+  if (!user) {
+    res.sendStatus(404);
+
+    return;
+  }
+
+  res.status(200).send(user);
+};
+
+const add = (req, res) => {
+  const { name } = req.body;
+
+  if (!name) {
+    res.sendStatus(400);
+
+    return;
+  }
+
+  const user = {
+    id: +new Date(),
```

Comment:
Again, I am not sure whether it is a good practice to add date as id. In the majority of cases it would be unique, but...